### ISSUE TITLE
No 118 and 119 based Thorium releases

### ISSUE THREAD

**Issue by nesper8** [Alex313031/thorium#430]
Its been a while that the latest release of Thorium has been a version based on 117 which is outdated by now
The latest version of chrome is version 119 with 120 on the horizon

Will there be any new releases or moving to a source only based release schedule?

**Comment by gz83** [comment]
Sorry, we will not be releasing a new version recently. The next version should follow Google's latest stable version.

**Comment by PF4Public** [comment]
@gz83 You mean Extended Stable?

**Comment by gz83** [comment]
> @gz83 You mean Extended Stable?

I'm sorry, that's not what I meant.

Please understand that the reason for not being able to release a new version of the browser at this time is that Alex needs to take care of some personal matters and will need to be away from GitHub for a while, during which time he will not be able to take care of things within GitHub or will be slower to take care of various things.

While I can help Alex with updating the repository and compiling new versions, it is against Alex's rules for me to update the repository myself.

@PF4Public

**Comment by xtarlit** [comment]
I feel as if the fact that one person is responsible for pushing all updates might be something that needs to be looked into at some point. Browsers get updates pretty often, and that usually comes with some sort of security patch or feature that probably shouldn't be delayed when coming to regular users. Thorium is a great browser, but it's sometimes hard to rely on when something like this happens where the only person allowed to push updates out is inactive. Ultimately it's up to Alex to decide what happens with the project, but it is slightly worrying considering the number of users Thorium has at this point.

**Comment by RoyRao2333** [comment]
I personally understand and respect what @gz83 said. And as much as I can tell, he had truly done a lot for this project by answering and resolving issues in this project.

But you must admit that @xtarlit is right. I cannot find a second Chromium based browser that fits my needs other than Thorium. I'm also using *ungoogled-chromium* because it's updating rather more frequently, but just for development as it not syncing my stuffs to Google.

> While I can help Alex with updating the repository and compiling new versions, it is against Alex's rules for me to update the repository myself.

I think one must take over this project while the owner is away, and users can PR to contribute. You may ask Alex for permission to publish a new release, but this cannot be delayed as it'll make you lose many users.

> Thorium is a great browser, but it's sometimes hard to rely on when something like this happens where the only person allowed to push updates out is inactive.

As they published v117 for Windows and Linux, I can only use v116 for macOS... (I refresh Thorium-MacOS repository once a day 🤣)

**Comment by PF4Public** [comment]
> _ungoogled-chromium_ because it's updating rather more frequently

This became possible only after moving ungoogled-chromium into its own org and allowing more people to contribute and take care of its advancement. I would suggest the same for Thorium.

**Comment by onegreyonewhite** [comment]
@Alex313031 It turns out the project is stagnating? It seems to me that the project lacks features and lacks regular updates from the main branch.

I really enjoy using this browser as my main one. But being two major releases behind looks like the project has been abandoned. Therefore, I would like to help somehow, but I see that the new PR does not have much effect.

**Comment by thisislvca** [comment]
Totally agree with @PF4Public . A Thorium org would be ideal considering how many people are using it right now. @gz83 Can you contact Alex off-github and suggest the idea/try to work something out? It really saddens me to see such a good browser not improving anymore just because the creator is not active for the time being.

I 100% understand and would do the same if I were in his situation, but for those who use the actual thing every day, it'd be nice to have a properly open-source structure that allows the project to be always maintained and updated, whatever happens. 

That's why open source is great :)

**Comment by alexsch01** [comment]
Chromium 120 was recently released

**Comment by gz83** [comment]
> Totally agree with @PF4Public . A Thorium org would be ideal considering how many people are using it right now. @gz83 Can you contact Alex off-github and suggest the idea/try to work something out? It really saddens me to see such a good browser not improving anymore just because the creator is not active for the time being.
> 
> I 100% understand and would do the same if I were in his situation, but for those who use the actual thing every day, it'd be nice to have a properly open-source structure that allows the project to be always maintained and updated, whatever happens.
> 
> That's why open source is great :)

I can't successfully contact Alex now. Alex told me before that he needs to leave for a while, so I will contact him again before the end of this month.  In addition, if I still can't contact Alex after December, I will consider getting involved in the management of the repo.

@thisislvca

**Comment by RoyRao2333** [comment]
> In addition, if I still can't contact Alex after December, I will consider getting involved in the management of the repo.

@gz83 Thanks for everything. 😊 

And, please don't forget about macOS users...

**Comment by earthsound** [comment]
> > Totally agree with @PF4Public . A Thorium org would be ideal considering how many people are using it right now. @gz83 Can you contact Alex off-github and suggest the idea/try to work something out? It really saddens me to see such a good browser not improving anymore just because the creator is not active for the time being.
> > I 100% understand and would do the same if I were in his situation, but for those who use the actual thing every day, it'd be nice to have a properly open-source structure that allows the project to be always maintained and updated, whatever happens.
> > That's why open source is great :)
> 
> I can't successfully contact Alex now. Alex told me before that he needs to leave for a while, so I will contact him again before the end of this month. In addition, if I still can't contact Alex after December, I will consider getting involved in the management of the repo.
> 
> @thisislvca

@gz83 Considering the very large number of unpatched high severity security vulnerabilities in the latest releases of all versions of Thorium, I would suggest considering repo involvement/management/releases now. 

[As I mentioned in another issue](https://github.com/Alex313031/thorium/issues/386#issuecomment-1854552607), just in November and December, stable releases of chrome have addressed 31 security issues most of them with a severity of high.

**Comment by PF4Public** [comment]
Come on people! Stop panicking! Please have some patience.

**Comment by thisislvca** [comment]
> Come on people! Stop panicking! Please have some patience.

The problem to me is I was using Thorium as my main browser, and since I need to have the latest version for work, using it just isn't an option anymore after such a long time with literally zero updates...

I do have patience, but I'd like to have an org where everybody can contribute at all times 😅

**Comment by PF4Public** [comment]
@thisislvca Well, you could've forked an open-source Thorium project and update it yourself. Why didn't you do so? Please don't tell me that you're waiting for "everybody" to "contribute at all times" for your enjoyment. If you want someone work _for you_ you pay — plain and simple.

**Comment by thisislvca** [comment]
> @thisislvca Well, you could've forked an open-source Thorium project and update it yourself. Why didn't you do so? Please don't tell me that you're waiting for "everybody" to "contribute at all times" for your enjoyment. If you want someone to work _for you_ you pay — plain and simple.

Lmao no need to lecture me. I don't have the competence and time to learn how to do it right now.

Secondly, the whole experience is falling apart. Websites just breaking out of nowhere, and video performance going to hell randomly (using way too much cpu or causing ram leakages).

Besides, I've already switched to chrome a while ago, and after issue #463 I'm probably never using this again.

**Comment by inmymum** [comment]
Will anyone ever address thoriumos. With no way to install another browser, should I just switch to chromiumos?

**Comment by thisislvca** [comment]
@inmymum  that's probably the best choice
